adminservice/src/main/java/nu/yona/server/AdminServiceApplication.java

```diff
@@ -30,7 +32,19 @@
 {
 	public static void main(String[] args)
 	{
-		SpringApplication.run(AdminServiceApplication.class, args);
+		try
+		{
+			SpringApplication.run(AdminServiceApplication.class, args);
+		}
+		catch (Exception ex)
+		{
+
+			// issue in Hazelcast: it doesn't shutdown automatically,
+			// while we want this for the short running database initializer
```

Comment:
Corrected it. It's only for exceptions. If the startup is successful, the call to ``SpringApplication.run`` will return, so if we would put the Hazelcast shutdown in a finally block, we would shut down Hazel cast immediately after service startup.

Hopefully, they fix hazelcast/hazelcast#6339 quickly. I will see whether I can create a little repro for them.